### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by darrenodi**
In this update , I 
-Added the Works section of the mobile portfolio
-Added the About me section of the mobile portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,18 +55,187 @@ <h2>Multi-Post Stories</h2>
...
 
+          <p>A daily selection of privately personalized reads;
+             no accounts or sign-ups required. has been the 
+             industry's standard</p> 
+
+          <div class="tags">
+            <div class="ltags">html</div>
+            <div class="ltags">bootstrap</div>
+            <div class="ltags">Ruby</div>
+          </div>
```
Comment (Emmyn5600) [inline index.html:73]: - [ ] This should be wrapped in the `ul tag`
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 2]
```diff
@@ -55,18 +55,187 @@ <h2>Multi-Post Stories</h2>
           <div class="ltags">bootstrap</div>
           <div class="ltags">Ruby</div>
         </div>
-        <button class="project">See Project</button>   
+        <button type="button" class="project-button">See Project</button>   
       </div>
 
       <div class="card-works">
```
Comment (Emmyn5600) [inline index.html:61]: - [ ] We need to wrap this div inside the `section` tag
The container of the section is a section tag

File: index.html [hunk 3]
```diff
@@ -55,18 +55,187 @@ <h2>Multi-Post Stories</h2>
...
+
+          <p>A daily selection of privately personalized reads;
+             no accounts or sign-ups required. has been the 
+             industry's standard</p> 
+
+          <div class="tags">
+            <div class="ltags">html</div>
+            <div class="ltags">bootstrap</div>
+            <div class="ltags">Ruby</div>
+          </div>
```
Comment (Emmyn5600) [inline index.html:91]: - [ ] This should be wrapped in the `ul tag`
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 4]
```diff
@@ -55,18 +55,187 @@ <h2>Multi-Post Stories</h2>
...
+          <h2>Profesional Art Printing Data</h2> 
+
+          <p>A daily selection of privately personalized reads;
+             no accounts or sign-ups required. has been the 
+             industry's standard</p> 
+
+          <div class="tags">
+            <div class="ltags">html</div>
+            <div class="ltags">bootstrap</div>
+            <div class="ltags">Ruby</div>
```
Comment (Emmyn5600) [inline index.html:108]: - [ ] This should be wrapped in the `ul tag`
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 5]
```diff
@@ -55,18 +55,187 @@ <h2>Multi-Post Stories</h2>
...
+          </div>
```
Comment (Emmyn5600) [inline index.html:127]: - [ ] This should be wrapped in the `ul tag`
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 6]
```diff
@@ -55,18 +55,187 @@ <h2>Multi-Post Stories</h2>
...
+          <h2>Profesional Art Printing Data</h2> 
+
+          <p>A daily selection of privately personalized reads;
+             no accounts or sign-ups required. has been the 
+             industry's standard</p> 
+
+          <div class="tags">
+            <div class="ltags">html</div>
+            <div class="ltags">bootstrap</div>
+            <div class="ltags">Ruby</div>
```
Comment (Emmyn5600) [inline index.html:144]: - [ ] This should be wrapped in the `ul tag`
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 7]
```diff
@@ -55,18 +55,187 @@ <h2>Multi-Post Stories</h2>
...
+          </div>
```
Comment (Emmyn5600) [inline index.html:163]: - [ ] This should be wrapped in the `ul tag`
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 8]
```diff
@@ -55,18 +55,187 @@ <h2>Multi-Post Stories</h2>
           <div class="ltags">bootstrap</div>
           <div class="ltags">Ruby</div>
         </div>
-        <button class="project">See Project</button>   
+        <button type="button" class="project-button">See Project</button>
```
Comment (Emmyn5600) [inline index.html:58]: Nice job here for the `see project` button however let us fix this issue
port17
The hover color you chose makes your button disappear once you pointed on it.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @darrenodi 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  

       * #### Please check the comment under the review
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 2 above)
Comment (darrenodi) [reply index.html:61]: Hello. I don't completely understand this point

Comment (Emmyn5600) [reply index.html:61]: Hi @darrenodi , you have to wrap the technology in the ul li tag, 
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

Comment (darrenodi) [reply index.html:61]: It's already in a section tag outside the div

**Comment by darrenodi** [comment]
All done

**Review by Emmyn5600 — APPROVED** [review]
Hi, 
Good job so far! 👏🏼 @darrenodi 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed, however use a descriptive feature branch:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by darrenodi** [comment]
Thank you :)

**Pull request merged**